### PR TITLE
Release v0.8.5

### PR DESCRIPTION
**Pull request by NemTechCN**
1. update fee amount ; 
2. add cosignature ; 
3. add transfer transactino unit test ; 
4. fix transfer transaction bug ; 
5. refactor routing  ; 
6. fix invoice page ; 
7. signTransaction modification ; 
8. disable forms when no network currency data  #488 ; 
9. refactor tx detail and make it scrollable ； 
10. press enter to submit forms ; 
11. fix other bugs.

### CODE REVIEW

File: __tests__/views/monitor/monitor-transfer/transactions/MonitorTransferForm.spec.ts [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+        expect(innerTransferTransaction.version).toBe(1)
+        expect(innerTransferTransaction.deadline instanceof Deadline).toBe(true)
+        expect(innerTransferTransaction.maxFee instanceof UInt64).toBe(true)
+        expect(innerTransferTransaction.recipientAddress instanceof Address).toBe(true)
+        innerTransferTransaction.mosaics.forEach(item => {
+            expect(item instanceof Mosaic).toBe(true)
+        })
+    })
+
+    it('should not create a aggregate bonded transaction object while choose 2-of-2 multisig', async() => {
```
Comment (evias) [inline __tests__/views/monitor/monitor-transfer/transactions/MonitorTransferForm.spec.ts]: typo on `not`. It **should* create a aggregate bonded for 2-of-2

File: __tests__/views/monitor/monitor-transfer/transactions/MonitorTransferForm.spec.ts [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+        expect(transferTransaction.version).toBe(1)
+        expect(transferTransaction.deadline instanceof Deadline).toBe(true)
+        expect(transferTransaction.maxFee instanceof UInt64).toBe(true)
+        expect(transferTransaction.recipientAddress instanceof Address).toBe(true)
+        transferTransaction.mosaics.forEach(item => {
+            expect(item instanceof Mosaic).toBe(true)
+        })
+    })
+
+    it('should not create a aggregate complete transaction object while choose 1-of-1 multisig', async () => {
```
Comment (evias) [inline __tests__/views/monitor/monitor-transfer/transactions/MonitorTransferForm.spec.ts]: typo on `no`. It *should create a aggregate complete for 1-of-1

File: __tests__/views/monitor/monitor-transfer/transactions/MonitorTransferForm.spec.ts (hunk 1 above)
Comment (NemTechCN) [reply __tests__/views/monitor/monitor-transfer/transactions/MonitorTransferForm.spec.ts]: fixed

File: __tests__/views/monitor/monitor-transfer/transactions/MonitorTransferForm.spec.ts (hunk 2 above)
Comment (NemTechCN) [reply __tests__/views/monitor/monitor-transfer/transactions/MonitorTransferForm.spec.ts]: fixed

**Pull request merged**